dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1895,22 +1895,24 @@ def  put_participant_in_section(student, teacher, script)
   end
 
   test 'should redirect to 2017 version in script family' do
-    cats1 = create :script, name: 'cats1', family_name: 'ui-test-versioned-script', version_year: '2017'
+    cats1 = create :script, name: 'cats1', family_name: 'cats', version_year: '2017', is_course: true
+    CourseOffering.add_course_offering(cats1)
 
     assert_raises ActiveRecord::RecordNotFound do
-      get :show, params: {script_id: 'ui-test-versioned-script', lesson_position: 1, id: 1}
+      get :show, params: {script_id: 'cats', lesson_position: 1, id: 1}
     end
 
     cats1.update!(published_state: Curriculum::SharedCourseConstants::PUBLISHED_STATE.stable)
-    get :show, params: {script_id: 'ui-test-versioned-script', lesson_position: 1, id: 1}
+    get :show, params: {script_id: 'cats', lesson_position: 1, id: 1}
     assert_redirected_to "/s/cats1/lessons/1/levels/1"
 
-    create :script, name: 'cats2', family_name: 'ui-test-versioned-script', version_year: '2018', published_state: Curriculum::SharedCourseConstants::PUBLISHED_STATE.stable
-    get :show, params: {script_id: 'ui-test-versioned-script', lesson_position: 1, id: 1}
+    cats2 = create :script, name: 'cats2', family_name: 'cats', version_year: '2018', is_course: true, published_state: Curriculum::SharedCourseConstants::PUBLISHED_STATE.stable
+    CourseOffering.add_course_offering(cats2)
+    get :show, params: {script_id: 'cats', lesson_position: 1, id: 1}
     assert_redirected_to "/s/cats2/lessons/1/levels/1"
 
     # next redirects to latest version in a script family
-    get :next, params: {script_id: 'ui-test-versioned-script'}
+    get :next, params: {script_id: 'cats'}
```

Comment:
Great cleanup –– this is much clearer to me, who still gets confused about family names!